config/routes.rb

```diff
@@ -6,6 +6,22 @@
   direct :homepage do "#{root_url}" end
 
   resources :home
+  resources :posts, :comments do
+    :comments
+  end
+  resources :home do
+    member do
+      get :get_comments
+    end
+  end
+  resources :posts do
+    member do
+      patch :destroy_post
```

Comment:
No need of this

Reply:
I wanted to create a separate method for 'destroy post', that's why